pex/bin/pex.py

```diff
@@ -565,7 +565,8 @@ def build_pex(args, options, resolver_option_builder):
                                 interpreters=interpreters,
                                 platforms=options.platform,
                                 cache=options.cache_dir,
-                                cache_ttl=options.cache_ttl)
+                                cache_ttl=options.cache_ttl,
+                                allow_prereleases=resolver_option_builder._allow_prereleases)
```

Comment:
actually, one quick ask: this access of `_allow_prereleases` violates the private convention. would you mind exposing `_allow_prereleases` here as a public attribute of the `ResolverOptionsBuilder` class and using that instead? a quick `@property def allow_prereleases(): return self._allow_prereleases` would work.

Reply:
Sure. That's easy. The only reason I did not do it initially is because I did not know what would be the code owner's preferences. I'll do it and resubmit.

Reply:
Actually, I just realized that there was another reason. The existing method:
```
  def allow_prereleases(self, allowed):
    self._allow_prereleases = allowed
    return self
```
is not exactly a property setter, because it returns `self`.
I'll try to work around the existing interface and see what feels the best.